compiler/AST/LoopStmt.cpp

```diff
@@ -79,9 +80,28 @@ void LoopStmt::setHasVectorizationHazard(bool v)
   mVectorizationHazard = v;
 }
 
+bool LoopStmt::hasParallelAccessVectorizationHazard() const
+{
+  return mParallelAccessVectorizationHazard;
+}
+
+void LoopStmt::setHasParallelAccessVectorizationHazard(bool v)
+{
+  mParallelAccessVectorizationHazard = v;
```

Comment:
It seems like right now `mParallelAccessVectorizationHazard` will always be false (false is the default value in line 30 above and I don't see a line like `setHasParallelAccessVectorizationHazard(true)` elsewhere). Are there cases where `mParallelAccessVectorizationHazard` will be true?

Reply:
@slnguyen - that's a good point. I added `mParallelAccessVectorizationHazard` because in the future we are going to need to distinguish between vectorization hazards for the LLVM vectorizer and hazards for RV. (At first I thought I would need it in this PR, but in the end I did not). 

Hazards for RV and the LLVM vectorizer will different, but currently the compiler doesn't know that. For example, following https://github.com/chapel-lang/chapel/issues/7761#issuecomment-669090509 we might allow RV to vectorize loops that have `in` intent on foralls to create task-ish-local-storage (and somehow create per-vector-lane storage for these in vectorized loops). This won't work for the LLVM vectorizer though, so `in` intent on a forall should remain a hazard for the LLVM vectorizer (which we communicate with by "parallel access" hints).